dev/io.openliberty.io.netty/src/io/netty/handler/timeout/WriteTimeoutHandler.java

```diff
@@ -0,0 +1,268 @@
+/*
```

Comment:
Just for my understanding this is an overlay directly from the Netty project with some specific changes we require, correct?